Update WinSDK.js to handle recursively translation files

fernandolguevara opened the pull request:
fix issue - https://crosswalk-project.org/jira/projects/XWALK/issues/XWALK-7420?filter=allopenissues

rakuco commented:
@darktears, @pozdnyakov, @baleboy: ping

@fernandolguevara: thanks for the patch. Can you be more descriptive in both the PR message and the commit message? Specifically, it would be good to explain what the problem was and what your patch does to fix it. Last but not least, it looks like you're using a different indentation than the rest of the code, so that needs to be fixed.

fernandolguevara commented:
@rakuco the problem is in the generation of wix file, the folder containing the translations also has sub folders with more translations files and the current code cant handle this, my patch fix that situation.

I found another issue , if you run the generation of msi file some times the new msi file contains the olders ...

rakuco commented:
I see. My point with updating your messages is that this sort of information should be part of the PR and, most importantly, the commit, so that people using `git log` or `git blame` years from now will still be able to figure out why a certain commit was merged without having to backtrack to GitHub and then JIRA.

Commit:
Update WinSDK.js to handle recursively translation files

fix issue - https://crosswalk-project.org/jira/projects/XWALK/issues/XWALK-7420?filter=allopenissues

fernandolguevara commented:
@rakuco done
